modules/standard/SysBasic.chpl

```diff
@@ -98,227 +175,757 @@ inline proc c_calloc(type eltType, size: integral) {
+*/
 extern const EINTR:err_t;
+
+/* Invalid argument.  Some invalid argument was supplied.  (For example,
+   specifying an undefined signal to a signal system call or a kill system
+   call). (POSIX.1)
+*/
 extern const EINVAL:err_t;
+
+/* Input/output error.  Some physical input or output error occurred.  This
+   error will not be reported until a subsequent operation on the same file
+   descriptor and may be lost (over written) by any subsequent errors. (POSIX.1)
+  */
 extern const EIO:err_t;
+
+/* Socket is already connected.  A connect system call was made on an already
+   connected socket; or, a sendto or sendmsg system call on a connected socket
+   specified a destination when already connected. (POSIX.1)
+*/
 extern const EISCONN:err_t;
+
+/* Is a directory. An attempt was made to open a directory with write mode
+   specified.  (POSIX.1)
+ */
 extern const EISDIR:err_t;
+
+/* Is a named type file (linux only) */
 extern const EISNAM:err_t;
+
+/* Key has expired (linux only) */
 extern const EKEYEXPIRED:err_t;
+
+/* Key was rejected by service (linux only) */
 extern const EKEYREJECTED:err_t;
+
+/* Key has been revoked (linux only) */
 extern const EKEYREVOKED:err_t;
+
+/* Level 2 halted (linux only) */
 extern const EL2HLT:err_t;
+
+/* Level 2 not synchronized (linux only) */
 extern const EL2NSYNC:err_t;
+
+/* Level 3 halted (linux only) */
 extern const EL3HLT:err_t;
+
+/* Level 3 halted (linux only) */
 extern const EL3RST:err_t;
+
+/* Cannot access a needed shared library (linux only) */
 extern const ELIBACC:err_t;
+
+/* Accessing a corrupted shared library (linux only) */
 extern const ELIBBAD:err_t;
+
+/* Attempting to link in too many shared libraries (linux only) */
 extern const ELIBMAX:err_t;
+
+/* lib section in a.out corrupted (linux only) */
 extern const ELIBSCN:err_t;
+
+/* Cannot exec a shared library directly (linux only) */
 extern const ELIBEXEC:err_t;
+
+/* Too many levels of symbolic links.  A path name lookup involved more than 32
+   (MAXSYMLINKS) symbolic links.  (POSIX.1)
+  */
 extern const ELOOP:err_t;
+
+/* Wrong medium type (linux only) */
 extern const EMEDIUMTYPE:err_t;
+
+/* Too many open files.  Maximum number of file descriptors allowable in the
+   process has been reached and requests for an open cannot be satisfied until
+   at least one has been closed.  The getdtablesize system call will obtain the
+   current limit. (POSIX.1)
+*/
 extern const EMFILE:err_t;
+
+/* Too many links. Maximum allowable hard links to a single file has been
+   exceeded. (POSIX.1)
+  */
 extern const EMLINK:err_t;
+
+/* Message too long.  A message sent on a socket was larger than the internal 
+   message buffer or some other network limit. (POSIX.1)
+ */
 extern const EMSGSIZE:err_t;
+
+/* Multihop attempted. (POSIX.1)
+ */
 extern const EMULTIHOP:err_t;
+
+/* File name too long.  A component of a path name exceeded {NAME_MAX} charac-
+   ters, or an entire path name exceeded {PATH_MAX} characters. (POSIX.1)
```

Comment:
I'd be tempted to move the entire word onto the next line, since that seems consistent with the rest of the file, but this is fine, too.